src/content/en/2021/css.md

```diff
@@ -158,7 +157,7 @@ Pages continue to use IDs, and at about the same rate as seen in 2020.  Even the
 ) }}
 
 We were unhappy to see so many IDs starting with `rc-`.  These are part of the reCAPTCHA system, which is inaccessible to the members of the deafblind community.
-
+{# TODO: (Authors) it would be a great to add a reference here to learn more, or to link to the a11y chapter's captcha section #}
```

Comment:
@meyerweb

Reply:
This is still being researched, and the paragraph may be removed entirely.  If not, there will be a link.

Reply:
found one link you can use: https://www.w3.org/TR/turingtest/

Reply:
This is resolved in the latest edits.